### PR TITLE
[#2118] Check for responseText in XHR callbacks to support logging in in...

### PR DESCRIPTION
**Pull request by cadecairos**
... IE9

Lighthouse Ticket: https://webmademovies.lighthouseapp.com/projects/65733/tickets/2118

### CODE REVIEW

File: src/cornfield/module.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ define(['util/xhr'], function(XHR) {
             function() {
               if (this.readyState === 4) {
                 try {
-                  var response = JSON.parse(this.response);
+                  var response = JSON.parse(this.response || this.responseText);
```
Comment (secretrobotron) [inline src/cornfield/module.js]: Spacing please!

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d16e31de143c52e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d16e31de143c52e/output.txt

Total script time: 0.83 mins

Server is running on http://pr887.butyr.org/templates/basic/
Run the Test Suite: http://pr887.butyr.org/test/

**Pull request closed without merging**